src/blog/truffle-v5-has-arrived.md

```diff
@@ -0,0 +1,102 @@
+We have released the most awaited version of Truffle just in time for the
+holidays! There are SO many exciting things to mention that will make your
+Truffle experience a rich one. Highlighted below are the new exciting things
+Truffle has to offer.
+
+[Download Truffle v5!](https://truffleframework.com/truffle)
+
+- **Bring your own compiler!!!!** You can now choose any solc-js version
```

Comment:
Should we consistently start all these sentences with capital letters OR lowercase?
In other words, it is inconsistent to start "You can now choose..." and then start the next section "the truffle-contract command..."